app/uk/gov/hmrc/incorporatedentityidentificationfrontend/utils/MessagesHelper.scala

```diff
@@ -29,17 +29,23 @@ object MessagesHelper {
 
   def amendMessagesWithLabelsFromJourneyConfig(initialMessages: Map[String, Map[String, String]],
                                                journeyConfig: JourneyConfig): Map[String, Map[String, String]] = {
+
     val extraEnglishTranslations: Map[String, String] = journeyConfig
       .pageConfig
-      .optServiceName
-      .map(serviceName => Map(optServiceNameTranslationKey -> serviceName))
-      .getOrElse(Map.empty)
+      .optLabels match {
+      case Some(JourneyLabels(_, Some(optEnglishServiceName))) => Map(optServiceNameTranslationKey -> optEnglishServiceName)
+      case Some(JourneyLabels(_, None)) | None => journeyConfig.pageConfig.optServiceName
```

Comment:
Do we need this case, if the serviceName isn't present would we just use the default anyway? For example, in TitleHelper we check to see optServiceName is present in the messages file and if it's not then we use the default in the messages file instead

Reply:
This case was to support situation when the calling service is not passing english labels but is still maybe using old way. I thought to include it to keep backward compatibility. Of course default will be used if nothing found in labels or in old optServiceName.

Reply:
Ahh yeah makes sense. Leave as is then